source/dem/dem.cc

```diff
@@ -292,6 +305,15 @@ DEMSolver<dim>::load_balance()
                                             cells_local_neighbor_list,
                                             cells_ghost_neighbor_list);
 
+  // Get total (with repetition) neighbors list for floating mesh
```

Comment:
Can you make this comment clear? I don't understand

Reply:
Added.